src/index.js

```diff
@@ -44,7 +44,13 @@ export default function babel ( options ) {
 		},
 
 		load ( id ) {
-			if ( id === HELPERS ) {
+			if ( id !== HELPERS ) {
+				return;
+			}
+
+			const babelsMajor = parseInt(babelVersion[0], 10);
+
+			if (babelsMajor < 7) {
```

Comment:
The main downside of this is that it'll break for the previous 7.x versions. Maybe we can just add a dep on `semver` and explicitly check that the version satisfies `>=7.0.0-beta.0`?

Reply:
thought about it, but gave up on this because:
- it would add complexity
- 7 is still beta :P i see such allowed peer dep usable only to satisfy npm and omit warnings
- there would be a gap of compatibility (what a phrase, lol) former versions would work with those regexps, but version which broke them wouldnt work at all and next ones wouldnt work too until version which will fix `outputType: "module"` (next one)

wdyt?

Reply:
I think there's probably very few people using both rollup-plugin-babel (99k downloads/month) + babel 7

Reply:
only one more argument not to do it in more complex way imho